### PR TITLE
Updated documentation for detailed processing log in KsqlDB.io project

### PR DESCRIPTION
**Pull request by deepak060793**
### Description 
ksql.logging.processing.rows.include can be changed in CC by user while provisioning new ksqldb

### Testing done 
Tested in UI. Attached below images.
image

image

Jira tickets:-
https://confluentinc.atlassian.net/browse/KCI-1715
https://confluentinc.atlassian.net/browse/KCI-1745

### Reviewer checklist
- [ X] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ X] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/reference/processing-log.md [hunk 1]
```diff
@@ -104,7 +104,9 @@ setting the ksqlDB property `ksql.logging.processing.rows.include` to
 !!! important
     In {{ site.ccloud }}, `ksql.logging.processing.rows.include` is set
     to `true`, so the default behavior is to include row data in the
-    processing log. Contact support to disable this default setting.
+    processing log. It can be configured manually when provisioning the cluster in 
+    Confluent Cloud by toggling the hide row data when provisioning
```
Comment (JimGalasyn) [inline docs/reference/processing-log.md:108]:
```suggestion
    {{ site.ccloud }} by toggling the hide row data when provisioning 
```

Comment (JimGalasyn) [inline docs/reference/processing-log.md:108]:
```suggestion
    Confluent Cloud by toggling **hide row data** when provisioning 
```

File: docs/reference/processing-log.md [hunk 2]
```diff
@@ -104,7 +104,9 @@ setting the ksqlDB property `ksql.logging.processing.rows.include` to
 !!! important
     In {{ site.ccloud }}, `ksql.logging.processing.rows.include` is set
     to `true`, so the default behavior is to include row data in the
-    processing log. Contact support to disable this default setting.
+    processing log. It can be configured manually when provisioning the cluster in 
+    Confluent Cloud by toggling the hide row data when provisioning 
+    in the UI, or setting the `log-exclude-rows` flag in the CLI
```
Comment (JimGalasyn) [inline docs/reference/processing-log.md:109]:
```suggestion
    in the UI, or by setting the `log-exclude-rows` flag in the CLI.
```

File: docs/reference/server-configuration.md [hunk 3]
```diff
@@ -375,7 +375,9 @@ messages. By default, this property has the value `false`.
 !!! important
     In {{ site.ccloud }}, `ksql.logging.processing.rows.include` is set
     to `true`, so the default behavior is to include row data in the
-    processing log. Contact support to disable this default setting.
+    processing log. It can be configured manually when provisioning the cluster in 
+    Confluent Cloud by toggling the hide row data when provisioning
```
Comment (JimGalasyn) [inline docs/reference/server-configuration.md:379]:
```suggestion
    {{ site.ccloud }} by toggling ** hide row data** when provisioning 
```

File: docs/reference/server-configuration.md [hunk 4]
```diff
@@ -375,7 +375,9 @@ messages. By default, this property has the value `false`.
 !!! important
     In {{ site.ccloud }}, `ksql.logging.processing.rows.include` is set
     to `true`, so the default behavior is to include row data in the
-    processing log. Contact support to disable this default setting.
+    processing log. It can be configured manually when provisioning the cluster in 
+    Confluent Cloud by toggling the hide row data when provisioning 
+    in the UI, or setting the `log-exclude-rows` flag in the CLI
```
Comment (JimGalasyn) [inline docs/reference/server-configuration.md:380]:
```suggestion
    in the UI, or by setting the `log-exclude-rows` flag in the CLI.
```

**Review by JimGalasyn — APPROVED** [review]
LGTM, with a few small suggestions. If you'd like this to go live now, I'll cherry-pick to the live docs branch. Thanks for the PR, @deepak060793!

**Comment by deepak060793** [comment]
> LGTM, with a few small suggestions. If you'd like this to go live now, I'll cherry-pick to the live docs branch. Thanks for the PR, @deepak060793!

Actually, I am waiting for the UI changes to be merge-in. Once, that will goes in, then I would like to make it live. Thanks for your comments.

**Comment by JimGalasyn** [comment]
@deepak060793 Sounds good!

**Review by JSeb225 — APPROVED** [review]
LGTM

**Pull request merged**